SU2_PY/SU2_Nastran/pysu2_nastran.py

```diff
@@ -774,12 +855,14 @@ def __temporalIteration(self,time):
       self.a += (1-self.alpha_f)/(1-self.alpha_m)*self.qddot
     else:
       if self.ImposedMotionToSet:
+        if self.Config["RESTART_SOL"] == "NO":                                   # If yes we already set it in the __setRestart function
+          self.timeStartCoupling = time
```

Comment:
```suggestion
        if self.Config["RESTART_SOL"] == "NO":
          # If yes we already set it in the __setRestart function
          self.timeStartCoupling = time

```

Reply:
I am not sure whether a comment 20 spaces away is the best thing todo, I would rather have it close to its partner lines

Reply:
agree